.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+package-lock.json
+package.json
```

Comment:
These files `package.json` and `package-lock.json` are needed with your project because they hold all of the dependencies that you have used, kindly remove both of them from `.gitignore` file